addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.xml

```diff
@@ -4,7 +4,7 @@
         <div class="o_ComposerSuggestedRecipient" t-att-title="ADD_AS_RECIPIENT_AND_FOLLOWER_REASON">
             <t t-if="suggestedRecipientInfo">
                 <div class="custom-control custom-checkbox">
-                    <input t-attf-id="{{ id }}_checkbox" class="custom-control-input" type="checkbox" t-att-checked="suggestedRecipientInfo.isSelected ? 'checked' : undefined" t-on-change="_onChangeCheckbox" t-ref="checkbox" />
+                    <input t-attf-id="{{ id }}_checkbox" class="custom-control-input" t-att-class="{'has-partner': suggestedRecipientInfo.partner}" type="checkbox" t-att-checked="suggestedRecipientInfo.isSelected ? 'checked' : undefined" t-on-change="_onChangeCheckbox" t-ref="checkbox" />
```

Comment:
I think we want modifiers to also be prefixed by `o`.

And in this case I'm not sure if we really want a modifier. Maybe this should be `o_ComposerSuggestedRecipient_partnerCheckBox` or something like that.

Or if we just want to use it as a signature in the tests, it could also be a `data-`.

Can be discussed with  @alexkuhn